### ISSUE TITLE
Sorting order of Sprint under Train in menu

### ISSUE THREAD

**Issue by invincibleJai** [openshiftio/openshift.io#4122]
I liked the view where we show all associated Sprints under a train along with dates, it is very useful.

What I observed is sprint's are not sorted i.e 154 appears before 155. I feel it would be a value add form User pov to have it as per date of creation or sprint number.

Feel free to close it if it's desired behaviour.

<img width="604" alt="screen shot 2018-08-03 at 11 31 40 am" src="https://user-images.githubusercontent.com/5129024/43626815-206d8ab4-9711-11e8-8c47-fd65783269e0.png">

**Comment by Preeticp** [comment]
+1 @invincibleJai I have pointed this out to the team as well. You will notice the alphabetical order of the train is disrupted too because of this. Sorting it by date would put Jonquil, Knapweed and lilac in order.

**Comment by nimishamukherjee** [comment]
@openshiftio/uxd-team can you recommend the ordering here.
Do we go by iteration start date?
If yes, how do we handle iterations without a start date as we do not enforce iteration start and end dates.

**Comment by christianvogt** [comment]
@Veethika could you provide guidance on the sort order

**Comment by serenamarie125** [comment]
@Veethika I agree that Iterations should be sorted alphabetically by name, do you agree?

**Comment by Veethika** [comment]
@christianvogt @serenamarie125 @invincibleJai  
The sorting for iteration should definitely be on the basis of 'Start date'. At a later stage we could provide an option to sort by dates or names, but as a default, it should always follow the start date.
**Why?**   There could be situations in which the names might not follow a sequence(alphabetical/numerical), as they are merely there for identification. It would then become difficult to search for an iteration on the basis of dates, which in fact is the primary purpose of having iterations.

**Comment by serenamarie125** [comment]
Can we do primary and secondary sort?  If so, I would say primary should be start date, and secondary is by name.  

Agreed as Veethika said, eventually we should allow the user to set the sort order.

**Comment by serenamarie125** [comment]
Based on a conversation held on 2019-Jan-04 including Dev, PM and UX, the UX team will not be providing support for planner items. Closing this issue since this was solely around gathering requirements.
